__tests__/app.test.js

```diff
@@ -433,3 +464,98 @@ describe("GET /api/articles/:article_id/comments", () => {
     });
   });
 });
+
+describe("POST api/articles/:article_id/comments", () => {
+  test("200: returns object of article with updated votes", () => {
+    return request(app)
+      .post(`/api/articles/1/comment`)
+      .send({
+        username: "icellusedkars",
+        body: "blah blah blah blah something newsy blah blah blah",
+      })
+      .expect(200)
+      .then(({ body }) => {
+        const comment = body.comment;
+
+        expect(comment).toEqual({
+          comment_id: expect.any(Number),
+          article_id: 1,
+          author: "icellusedkars",
+          body: "blah blah blah blah something newsy blah blah blah",
+          created_at: expect.any(String),
+          votes: 0,
+        });
+      });
+  });
+
```

Comment:
You could add another 201 test to check that any additional properties in the request body are ignored. For example if we have a key of votes we would expect to create a new comment with the author and body but we would still expect votes to be 0.